deployment/media-insights-dataplane-streaming-stack.template

```diff
@@ -44,6 +44,8 @@ Resources:
         rules_to_suppress:
           - id: W89
             reason: "This Lambda function does not need to access any resource provisioned within a VPC."
+          - id: W92
```

Comment:
FYI, I'm fixing a few cfn nag issues in this PR, but these changes are unrelated to the boto3 user agent.